sdk/core/core/src/az_http_response_parser.c

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+#include "_az_span.h"
```

Comment:
I don't really like `"..."`. It's an unclear feature in C which brings almost no value.

Reply:
we can't use <header.h>  without adding the /src folder to the target with cmake.  
This is IMO, the advantage of using double quote. No need to add the directory.
And it also looks very clear that it is a private header like this.  If we ever have a problem with this, we can consider making the cmake changes